client/components/foldable-card/README.md

```diff
@@ -42,3 +42,4 @@ render: function() {
 * `onOpen`: function to be executed in addition to the expand action when the card is opened
 * `summary`: string or component to show next to the action button when closed
 * `clickableHeader`: boolean indicating if the whole header can be clicked to open the card
+* `disableToggle`: boolean indicating if the chevron up/down button can toggle the card. This is useful for example if you've expanded the card and want to prevent users from closing it unless some action inside the card is completed.
```

Comment:
Do we have a true need for this? Why wouldn't we want someone to close the card?

Reply:
This was [discussed in](https://github.com/Automattic/wp-calypso/pull/18722#issuecomment-335787750) so the user doesn't lose track that there's a pending confirmation dialog.